vendor/file.te

```diff
@@ -23,6 +23,7 @@ type sysfs_system_sleep_stats, sysfs_type, fs_type;
 type sysfs_timestamp_switch, sysfs_type, fs_type;
 type sysfs_tof_sensor, fs_type, sysfs_type;
 type sysfs_usb_supply, sysfs_type, fs_type;
+type sysfs_glove_mode, sysfs_type, fs_type;
```

Comment:
Mamma mia, my nice alphabetical ordering.. ;)

Reply:
Too easy to forget, oops :sweat_smile: